package.json

```diff
@@ -12,14 +12,14 @@
     "build": "npm-run-all --silent clean transpile transpile:match copy-typescript-definition size",
     "transpile": "rollup -c --environment FORMAT:umd && rollup -c --environment FORMAT:es",
     "transpile:match": "babel src/match.js -o match.js -f umd",
-    "size": "size=$(gzip-size $npm_package_main) && echo \"gzip size: $size / $(pretty-bytes $size)\"",
+    "size": "node -e \"process.stdout.write('gzip size: ')\" && cross-env gzip-size --raw dist/preact-router.js",
```

Comment:
Can we use `cross-env-shell` here too?

Reply:
Sadly not, it doesn't implement subshells just variable replacement. I think 34af87626570719aed6fc32382ca30bc63675c9d and 950bbe5f6106fa085658eab1445ceff50b3d856f were not meant to be included in this PR however since the former changes the package name and the latter is #284

Reply:
That's correct, sorry about that!